Source/Serilog.Exceptions/Filters/ExceptionFilterIgnoringByName.cs

```diff
@@ -0,0 +1,42 @@
+namespace Serilog.Exceptions.Filters
+{
+    using System;
+
+    /// <summary>
+    /// Filters the exception properties based only on their name. If exception property
+    /// matches any of provided property names, exception property is ignored altogether.
+    /// Comparison method is exact case-sensitive.
+    /// </summary>
+    public class ExceptionFilterIgnoringByName : IExceptionPropertyFilter
```

Comment:
I suggest reversing the naming to `IgnorePropertyByNameExceptionFilter` or perhaps `IgnoreNamedPropertyExceptionFilter`.

Reply:
Done